.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  build_css:
+    runs-on: ubuntu-latest
+    env:
+      source: public/dist/assets
+      destination: assets/website-2021/pr-${{ github.event.number }}/
```

Comment:
Should we create a folder for all of them, rather than putting them directly under website-2021?

Reply:
we could use `assets/website-2021-dev/` to avoid mixing prod and dev assets.